### PR TITLE
Wireframe feature

### PR DESCRIPTION
**Pull request by fobadara**
## Awesome Books: Wireframe

In this version we:

🖌️  created the basic skeleton for the application 

🖌️  ensured that it takes in title and author names

🖌️  ensured that it saves these details in the local storage

### CODE REVIEW

File: javascript/index.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  for (let index = 0; index < storedBooks.length; index += 1) {
+    if ((this.parentElement.children[0].innerText === storedBooks[index].title)
+      && (this.parentElement.children[1].innerText === storedBooks[index].author)) {
+      storedBooks.splice(index, 1);
+      localStorage.setItem('books', JSON.stringify(storedBooks));
+      storedBooks = JSON.parse(localStorage.getItem('books'));
+      window.location.reload();
+    }
+  }
+}
```
Comment (oxenprogrammer) [inline javascript/index.js]: **Changes Required**
Good effort on implementing the remove function, however;
- [ ] Currently your book object doesn't have a unique attribute for instance `id`. When I add more than one book with the same title, the `remove function` doesn't work or it doesn't remove the exact book object. Let us try and fix this bug.

**Review by oxenprogrammer — CHANGES_REQUESTED** [review]
Hi @fobadara and @Kingstalux , 

Good job so far!
**To Highlight**
- [x] Professional README
- [x] Git flow Followed.
- [x] LocalStorage working.
- [x] Add Book working
  
I can see you're really putting effort to become a world-class software developer :+1:
There are some issues that you still need to address in order to move forward to the next project.
Trust me you got this! :muscle:


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Consider the optional changes nice to have features and only work on them when you have satisfied the required changes_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: javascript/index.js (hunk 1 above)
Comment (fobadara) [reply javascript/index.js]: Thanks for the observation. We decided to tackle the root of the problem. we ensured that the same books is not repeated in the catalog.

**Review by ZawadiSumba66 — APPROVED** [review]
### Hi Team 😃 , 
### STATUS: APPROVED 👍 
Great job implementing the changes requested by the previous reviewer. 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**